internal/filedata/archive_manager_test.go

```diff
@@ -186,7 +186,7 @@ func TestFileUpdatedWithInvalidDataAndDoesNotBecomeValid(t *testing.T) {
 
 		writeMalformedArchive(p.filePath)
```

Comment:
TODO: It's unclear to me whether these should be errors or warnings. In Eli's spike of this functionality, there is extra logic to determine if the file once existed and then was removed - which might not be considered an error. 

This could be nice but it's more logic in the loop.

Reply:
If you are using relay proxy is deleting a file a valid thing to do, and what is the expected outcome of it?

Reply:
It's valid in the sense that you can do it and nothing bad will happen in that exact moment, since the environments are already loaded into memory. 

But I'd say it's not an expected thing, and not recommended, since it means that if Relay crashes or restarts, it's going to fail.